aten/src/ATen/native/Itertools.cpp

```diff
@@ -51,7 +51,7 @@ Tensor combinations(const Tensor& self, int64_t r, bool with_replacement) {
   std::vector<Tensor> grids = at::meshgrid(std::vector<Tensor>(r, self));
   Tensor mask = _triu_mask(num_elements, r, with_replacement, self.options());
   for(Tensor &t : grids) {
-    t = t.masked_select(mask);
+    t = t.masked_select(mask.to(at::kByte));
```

Comment:
Going to fix this later?

Reply:
I'm going to call this Note "masked_select does not accept bool yet" and reference this at other locations I see it

Reply:
Wouldn't it make sense to first do all non-BC breaking bool changes first (e.g., make `masked_select` accept bool along with byte), before making the BC-breaking bool changes (making comparison ops return bool)